internal/pkg/local/load_test.go

```diff
@@ -68,82 +62,3 @@ func TestLocalLoadModelNotFound(t *testing.T) {
 	assert.Error(t, err)
 	assert.Equal(t, "kind \"test\" not found", err.Error())
 }
-
-func TestLocalLoadModelInvalidTransformation(t *testing.T) {
```

Comment:
Tento test je v balicku `local`, ale ma byt v `mapper/transformation`, tak som ho presunul, aby nevnikali cyklicke zavislosti medzi balickami.

Reply:
Tento test vznikol preto, .... pretoze ak bol nevalidny `config.json` v transformacii, ... tak sme v `config.Content` nechali `nil` hodnotu a spadlo to. Takze tento test overuje, ze aj ked je `config.json` nevalidny, tak sa vrati chyba a nie `panic`.